enterprise-beans/4.0/apidocs/index.html

```diff
@@ -1,181 +1,76 @@
-<!DOCTYPE HTML>
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Frameset//EN" "http://www.w3.org/TR/html4/frameset.dtd">
 <!-- NewPage -->
 <html lang="en">
 <head>
-<!-- Generated by javadoc (11.0.7) on Fri Jun 05 21:12:37 IST 2020 -->
-<title>Overview (Jakarta Enterprise Beans 4.0.0-RC2 API)</title>
+<!-- Generated by javadoc (1.8.0_202) on Thu Oct 08 16:21:41 GMT 2020 -->
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-<meta name="dc.created" content="2020-06-05">
-<link rel="stylesheet" type="text/css" href="stylesheet.css" title="Style">
-<link rel="stylesheet" type="text/css" href="jquery/jquery-ui.css" title="Style">
-<script type="text/javascript" src="script.js"></script>
-<script type="text/javascript" src="jquery/jszip/dist/jszip.min.js"></script>
-<script type="text/javascript" src="jquery/jszip-utils/dist/jszip-utils.min.js"></script>
-<!--[if IE]>
-<script type="text/javascript" src="jquery/jszip-utils/dist/jszip-utils-ie.min.js"></script>
-<![endif]-->
-<script type="text/javascript" src="jquery/jquery-3.3.1.js"></script>
-<script type="text/javascript" src="jquery/jquery-migrate-3.0.1.js"></script>
-<script type="text/javascript" src="jquery/jquery-ui.js"></script>
-</head>
-<body>
-<script type="text/javascript"><!--
-    try {
-        if (location.href.indexOf('is-external=true') == -1) {
-            parent.document.title="Overview (Jakarta Enterprise Beans 4.0.0-RC2 API)";
+<title>Jakarta Enterprise Beans 4.0.0 API</title>
+<script type="text/javascript">
+    tmpTargetPage = "" + window.location.search;
+    if (tmpTargetPage != "" && tmpTargetPage != "undefined")
```

Comment:
Should the enterprise-beans/4.0/apidocs/index.html contain javascript?  Or would html be better?

Reply:
I guess it doesn't affect the displaying of the javadoc, does it?  At least I couldn't find a visible issue with the javadoc...

Reply:
I picked up the result of the api javadoc jar.
But I can try to generate again and see?

Reply:
We don't have a "no javascript" rule, so I'm not sure as to why this is relevant.

Reply:
It displays correctly for me also (using Firefox).  I mentioned it in case it slipped by without notice as unlikely as that was.

Reply:
> I picked up the result of the api javadoc jar.
> But I can try to generate again and see?

I tried generating locally and see the same javascript (which displayed correctly in firefox).